examples/create-window/main.go

```diff
@@ -17,6 +17,7 @@ func main() {
 		fmt.Println(err)
 		return
 	}
+	defer X.Close()
```

Comment:
Also put an `X.Close()` in there for good measure. What do you think?

Reply:
Yeah, this is a good practice.